save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -119,9 +119,10 @@ class OrganizationService(
 
     /**
      * @param name
+     * @param status
      * @return organization by name
      */
-    fun findByName(name: String) = organizationRepository.findByName(name)
+    fun findByName(name: String, status: OrganizationStatus = OrganizationStatus.CREATED) = organizationRepository.findByName(name)?.takeIf { it.status == status }
```

Comment:
maybe add method `findByNameAndStatus` in repository instead of filtering on application side?

Reply:
Done